src/skills-builder/skills-builder-modal/view-results/data/constants.js

```diff
@@ -0,0 +1,11 @@
+const COURSE = 'course';
+
+/* The below strings can be used to demonstrate how we are able to incorporate other lines of business
+const BOOT_CAMP = 'boot camp';
+const EXECUTIVE_EDUCATION = 'executive education';
+*/
+
+// eslint-disable-next-line import/prefer-default-export
+export const LOB = [
```

Comment:
Nit: I'd prefer constant names that doesn't use acronyms. I know what "LOB" is, but would prefer it to be `lineOfBusiness` or even something more generic like "productType".